contracts/RegistryAdapters/IronBankRegistryAdapter.sol

```diff
@@ -583,18 +633,23 @@ contract RegistryAdapterIronBank is Ownable {
         IUnitroller.Market memory market =
             IUnitroller(comptrollerAddress).markets(assetAddress);
 
-        uint256 balance = assetBalance(assetAddress);
         TokenAmount memory underlyingTokenBalance =
-            tokenAmount(balance, tokenAddress, tokenPriceUsdc);
+            tokenAmount(assetBalance(assetAddress), tokenAddress, tokenPriceUsdc);
```

Comment:
Was this change made due to stack depth issues or is this purely stylistic?

Reply:
Stack too deep issue